djmp/models.py

```diff
@@ -56,7 +54,7 @@ class Tileset(models.Model):
     cache_type = models.CharField(max_length=10, choices=CACHE_TYPES)
     # file cache params
     directory_layout = models.CharField(max_length=20, choices=DIR_LAYOUTS, blank=True, null=True)
-    directory = models.CharField(max_length=256, default=FILE_CACHE_DIRECTORY, blank=True, null=True)
```

Comment:
👍 I just ran into this on some stuff I was working on and it took me _forever_ to figure out why `override_settings` wasn't working in my unit tests.  Nice catch!

We should probably audit the codebase for usages of settings that aren't using `django.conf.settings`